docs/docs-beta/docs/integrations/libraries/fivetran.md

```diff
@@ -18,19 +18,65 @@ tags: [dagster-supported, etl]
 ---
 
 
+# Using Dagster with Fivetran
 
-The Dagster-Fivetran integration enables you to orchestrate data ingestion as part of a larger pipeline. Programmatically interact with the Fivetran REST API to initiate syncs and monitor their progress.
+This guide provides instructions for using Dagster with Fivetran using the `dagster-fivetran` library. Your Fivetran connector tables can be represented as assets in the Dagster asset graph, allowing you to track lineage and dependencies between Fivetran assets and data assets you are already modeling in Dagster. You can also use Dagster to orchestrate Fivetran connectors, allowing you to trigger syncs for these on a cadence or based on upstream data changes.
 
-### Installation
+## What you'll learn
 
-```bash
-pip install dagster-fivetran
-```
+- How to represent Fivetran assets in the Dagster asset graph, including lineage to other Dagster assets.
+- How to customize asset definition metadata for these Fivetran assets.
+- How to materialize Fivetran connector tables from Dagster.
+- How to customize how Fivetran connector tables are materialized.
 
-### Example
+<details>
+  <summary>Prerequisites</summary>
 
-<CodeExample filePath="integrations/fivetran.py" language="python" />
+- The `dagster` and `dagster-fivetran` libraries installed in your environment
+- Familiarity with asset definitions and the Dagster asset graph
+- Familiarity with Dagster resources
+- Familiarity with Fivetran concepts, like connectors and connector tables
+- A Fivetran workspace
+- A Fivetran API key and API secret. For more information, see [Getting Started](https://fivetran.com/docs/rest-api/getting-started) in the Fivetran REST API documentation.
+
+</details>
+
+## Represent Fivetran assets in the asset graph
+
+To load Fivetran assets into the Dagster asset graph, you must first construct a <PyObject module="dagster_fivetran" object="FivetranWorkspace" /> resource, which allows Dagster to communicate with your Fivetran workspace. You'll need to supply your account ID, API key and API secret. See [Getting Started](https://fivetran.com/docs/rest-api/getting-started) in the Fivetran REST API documentation for more information on how to create your API key and API secret.
```

Comment:
The `PyObject` components are creating an _undefined_. Is that specific to Fivetran or more broadly an issue with the new docs? If more broad we can merge and make a follow-up ticket to fix this.
<img width="351" alt="image" src="https://github.com/user-attachments/assets/8f8d747e-f1d8-4365-a11c-99ddeef5ed60" />

Reply:
I get the same PyObject not implemented yet error in the I/O manager guide that is already in the docs

In the I/O manager guide:
<img width="877" alt="Screenshot 2024-12-16 at 2 42 04 PM" src="https://github.com/user-attachments/assets/95cbdddd-a5d6-4e37-925d-225984e7a780" />

In the Fivetran docs:
<img width="938" alt="Screenshot 2024-12-16 at 2 43 17 PM" src="https://github.com/user-attachments/assets/5c4f4415-5cd1-4ee3-a412-b96d8ee7ebb5" />

Reply:
cc @neverett

Reply:
Yeah, I agree with @cmpadden -- let's merge this, and I'll make a ticket to follow up on `PyObject` implementation in new docs.